met-api/tests/unit/api/test_survey_translation.py

```diff
@@ -0,0 +1,105 @@
+"""Tests to verify the SurveyTranslation API endpoints.
+
+Test-Suite to ensure that the SurveyTranslation API endpoints are working as expected.
```

Comment:
Do we know why most of these files have two descriptions?

Reply:
Not sure. Followed the other test files. It can be removed.